Index: src/store/branches/graphs/contracts/state.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/store/branches/graphs/contracts/state.ts b/src/store/branches/graphs/contracts/state.ts
--- a/src/store/branches/graphs/contracts/state.ts	(revision 0f99c9bddf6273df638c81fc364df7dcbae9e70f)
+++ b/src/store/branches/graphs/contracts/state.ts	(date 1626600455070)
@@ -1,5 +1,12 @@
 import {LoadingStatus} from "../../../types";
 
+export enum GraphsTimescale {
+    hour= "hour",
+    day = "day",
+    week = "week",
+    month = "month"
+}
+
 export interface FetchGraphs {
     device_id: number
     timescale: string
Index: src/pages/SensorGraphs/index.tsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/pages/SensorGraphs/index.tsx b/src/pages/SensorGraphs/index.tsx
--- a/src/pages/SensorGraphs/index.tsx	(revision 0f99c9bddf6273df638c81fc364df7dcbae9e70f)
+++ b/src/pages/SensorGraphs/index.tsx	(date 1626601043535)
@@ -1,22 +1,27 @@
-import React, { useEffect } from "react";
-import {useDispatch, useSelector} from "react-redux";
+import React, {useEffect} from "react"
+import {useDispatch, useSelector} from "react-redux"
 
-import { selectGraphsState } from "../../store/branches/graphs/selectors";
-import { fetchGraphsDataAC } from "../../store/branches/graphs/actionCreators";
+import {selectGraphsState} from "../../store/branches/graphs/selectors"
+import {fetchGraphsDataAC} from "../../store/branches/graphs/actionCreators"
 
-import HeaderSensorGraphs from "./components/HeaderSensorGraphs";
-import SensorGraph from "../../components/GraphsChart/SensorGraph";
+import HeaderSensorGraphs from "./components/HeaderSensorGraphs"
+import SensorGraph from "../../components/GraphsChart/SensorGraph"
+import {WinStorage} from "../../services/AuthSrorage"
+import {GraphsTimescale} from "../../store/branches/graphs/contracts/state"
 
-import classes from "./SensorGraphs.module.scss";
+import classes from "./SensorGraphs.module.scss"
 
 
 const SensorGraphs: React.FC = () => {
     const dispatch = useDispatch()
     const {data, status} = useSelector(selectGraphsState)
+    const currentDevice = WinStorage.getDevice()
 
     useEffect(() => {
-        dispatch(fetchGraphsDataAC({device_id: 41, timescale: "day"}))
-    }, [dispatch])
+        if (!data) {
+            dispatch(fetchGraphsDataAC({device_id: +currentDevice, timescale: GraphsTimescale.day}))
+        }
+    }, [dispatch, currentDevice, data])
 
 
     return (
@@ -27,7 +32,7 @@
                 <SensorGraph sensorsGraphData={data} sensorsGraphDataLoading={status}/>
             </div>
         </div>
-    );
-};
+    )
+}
 
-export default SensorGraphs;
+export default SensorGraphs
Index: src/pages/SensorGraphs/SensorGraphs.module.scss
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/pages/SensorGraphs/SensorGraphs.module.scss b/src/pages/SensorGraphs/SensorGraphs.module.scss
--- a/src/pages/SensorGraphs/SensorGraphs.module.scss	(revision 0f99c9bddf6273df638c81fc364df7dcbae9e70f)
+++ b/src/pages/SensorGraphs/SensorGraphs.module.scss	(date 1626600661749)
@@ -1,10 +1,20 @@
 .SensorGraphsWrap {
 
+
+
   .mapBlock {
     display: flex;
     flex-wrap: wrap;
     justify-content: center;
-    max-width: 85%;
+  }
+}
+
+.linkWrap {
+
+  a {
+    text-decoration-line: none !important;
+    text-transform: none !important;
+    color: black !important;
   }
 }
 
Index: src/pages/SensorGraphs/components/HeaderSensorGraphs.tsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/pages/SensorGraphs/components/HeaderSensorGraphs.tsx b/src/pages/SensorGraphs/components/HeaderSensorGraphs.tsx
--- a/src/pages/SensorGraphs/components/HeaderSensorGraphs.tsx	(revision 0f99c9bddf6273df638c81fc364df7dcbae9e70f)
+++ b/src/pages/SensorGraphs/components/HeaderSensorGraphs.tsx	(date 1626599951542)
@@ -2,26 +2,29 @@
 import {Typography} from "antd"
 import SettingPopup from "../../../components/Setting"
 import {Link} from "react-router-dom"
+import clsx from "clsx"
+
+import {useCurrentSelection} from "../../../helpers/useCurrentSelection"
 
 import classes from "../SensorGraphs.module.scss"
 
 const {Title} = Typography
 
 const HeaderSensorGraphs: React.FC = () => {
+    const {device, site} = useCurrentSelection()
+
     return (
         <React.Fragment>
-            <div className="block-title">
-                <Title level={5} className="header-link">
-                    <Link to="/sites">4B COMPONENTS / </Link>
-                    <Link to="/devices"> GRAINE TERMINAL</Link>
-                    <Link to="/devices"> / JETTY</Link>
-                    / CON7 LANDSIDE-UPSTREAM ALIGNMENT #1-1
+            <div className={clsx("block-title", classes.linkWrap)}>
+                <Title level={5}>
+                    <Link to="/sites">{site?.title || "Location"} / </Link> {device?.title || "Devices"}
                 </Title>
+
             </div>
 
             <div className="d-flex-100">
                 <div className={classes.headerTitle}>
-                    <Title className="title-fs-24" level={2}>CON7 Landside-Upstream Alignment #1-1 - Live Graphs</Title>
+                    <Title level={2} className="title-fs-24">{device?.title || "Devices"} - Live Graphs</Title>
                 </div>
 
                 <SettingPopup/>
